prow/spyglass/spyglasstests/podlogartifact_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spyglasstests
+
+import (
+	"bytes"
+	"testing"
+
+	"k8s.io/test-infra/prow/spyglass"
+)
+
+func TestPodLogReadAll(t *testing.T) {
+	testCases := []struct {
+		name     string
+		artifact *spyglass.PodLogArtifact
+		expected []byte
+	}{
+		{
+			name:     "Job Podlog readall",
+			artifact: spyglass.NewPodLogArtifact("job", "123", "", fakeJa),
+			expected: []byte("clusterA"),
+		},
+		{
+			name:     "Jib Podlog readall",
```

Comment:
It's the same test as the one used by the pod log endpoint - I wanted to keep the tests as close as possible because they are essentially the same feature, but yes the naming can change